kornia/augmentation/mix_augmentation.py

```diff
@@ -99,7 +100,16 @@ def generate_parameters(self, batch_shape: torch.Size) -> Dict[str, torch.Tensor
 
     def apply_transform(self, input: torch.Tensor, label: torch.Tensor,  # type: ignore
                         params: Dict[str, torch.Tensor]) -> Tuple[torch.Tensor, torch.Tensor]:  # type: ignore
```

Comment:
Why there’s type ignore ?

Reply:
It is overridding the parent class. LIke what we did for forword.